src/Layer/FeatureGeometryLayer.js

```diff
@@ -0,0 +1,41 @@
+import { Group } from 'three';
+
+import GeometryLayer from 'Layer/GeometryLayer';
+import FeatureProcessing from 'Process/FeatureProcessing';
+import Feature2Mesh from 'Converter/Feature2Mesh';
+
+/**
+ * `FeatureGeometryLayer` is GeometryLayer.
+ * It allows to display features (geojson, kml) in 3D.
+ *
+ * @property {boolean} isFeatureGeometryLayer - Used to checkout whether this layer is
+ * a FeatureGeometryLayer. Default is true. You should not change this, as it is used
+ * internally for optimisation.
+ */
+class FeatureGeometryLayer extends GeometryLayer {
+    /**
+     * @constructor
+     * @extends GeometryLayer
+     *
+     * @param {string} id - The id of the layer, that should be unique. It is
+     * not mandatory, but an error will be emitted if this layer is added a
+     * {@link View} that already has a layer going by that id.
+     * @param {Object} [config] - Optional configuration, all elements in it
+     * will be merged as is in the layer.
+     * @param {function} [options.batchId] - optional function to create batchId attribute.
+     * It is passed the feature property and the feature index.
+     * As the batchId is using an unsigned int structure on 32 bits, the batchId could be between 0 and 4,294,967,295.
+     *
+     */
+    constructor(id, config = {}) {
+        config.update = FeatureProcessing.update;
+        config.convert = Feature2Mesh.convert({
+            batchId: config.batchId,
+        },
+        );
+        super(id, new Group(), config);
```

Comment:
Maybe you could add the `Group` parameter to the `config` ?

Reply:
yes